topic/timeseries/dask-weather-data-import.ipynb

```diff
@@ -582,8 +623,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Uncomment the following lines to process the actual weather data,\n",
-    "# we commented them out in order to avoid long-running operations\n",
+    "# Uncomment the following lines to process the actual weather data.\n",
+    "# The have been disabled in order to avoid long-running operations.\n",
```

Comment:
```suggestion
    "# Uncomment the following lines to process the actual weather data.\n",
    "# They have been disabled in order to avoid long-running operations.\n",
```